### ISSUE TITLE
Certain places are not visible

### ISSUE THREAD

**Issue by skbly7** [mrinaldhar/CabShare#11]
While doing the auto complete search, most important place like "RGIT Airport", etc are also not visible.

We should consider removing the **geocode** from type in auto complete drop down.

(sending pull request for the same)

**Comment by mrinaldhar** [comment]
I know, but does that really help this issue? It's still messy about getting the right entry on things like "Airport". 

Just "Airport" should be enough for getting to RGIT. :/

**Comment by skbly7** [comment]
@mrinaldhar No, by just typing Airport, you will get "Begumpet Airport", instead of "RGIT" one. Try once. :smile:

**Comment by mrinaldhar** [comment]
Yes,I wrote "should". :P This is what I want it to do, not what its doing.

**Comment by skbly7** [comment]
Yups, I understood your point.

Thing was even after writing "Rajiv Gandhi", I was unable to get Airport in existing implementation.
Here are the screenshots (before & then after)

Before
![Before](https://cloud.githubusercontent.com/assets/3490586/13903049/1ff5cbbe-ee8f-11e5-873e-59d1f0911233.png)

After
![After](https://cloud.githubusercontent.com/assets/3490586/13903052/294de19c-ee8f-11e5-8e47-21a2991af177.png)

**Comment by mrinaldhar** [comment]
Yeah, it does make search better, so merging that one. Let's keep this issue open until Search becomes really good.
